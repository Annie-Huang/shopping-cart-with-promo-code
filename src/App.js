import React from 'react';
import './App.css';
// !!!! Don't ever use
// import { ProductList } from "./components/ProductList/ProductList";
// It will failed the build, we need the default one that got connect, not the named one use for testing.
import ProductList from "./components/ProductList";

function App() {
    return (
        <div className="container-fluid">
            <h1>Please select your products from below...</h1>
            <ProductList />
        </div>
    );
}

export default App;
