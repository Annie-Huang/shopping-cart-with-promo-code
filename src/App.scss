// import starting from the src/ folder
@import "scss/variables.scss";

// can also be relative import
// @import "./variables.scss";

// we can use the $primaryColor variable now
h1, h2 {
  color: $primaryColor;
}
