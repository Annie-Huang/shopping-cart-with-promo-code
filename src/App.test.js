import React from 'react';
import App from './App';
import {shallow} from "enzyme";
import toJson from 'enzyme-to-json';

describe('App presentation', () => {
    it('should render App component', () => {
        const result = shallow(<App />);
        expect(toJson(result)).toMatchSnapshot();
    });
});
