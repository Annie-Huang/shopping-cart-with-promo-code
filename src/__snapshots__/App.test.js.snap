// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App presentation should render App component 1`] = `
<div
  className="container-fluid"
>
  <h1>
    Please select your products from below...
  </h1>
  <div
    className="scss-warning"
  >
    Test SASS Configure
  </div>
  <Connect(ProductList) />
  <Connect(ShoppingCart) />
</div>
`;
